flutter/digit-ui-components/digit_components/test/theme/theme_test.dart

```diff
@@ -0,0 +1,144 @@
+import 'package:digit_components/digit_components.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+void main() {
+  TestWidgetsFlutterBinding.ensureInitialized();
+  DigitUi.instance.initThemeComponents();
+
+  group('Digit theme', () {
+    group('Typography', () {
+      final digitTheme = DigitTheme.instance;
+
+      final widget = MaterialApp(
+        theme: digitTheme.mobileTheme,
+        home: Scaffold(
+          body: Builder(builder: (context) {
+            final theme = Theme.of(context).textTheme;
+            return Column(
+              children: [
+                Text('Headline XL', style: theme.displayMedium),
+                Text('Headline L', style: theme.headlineLarge),
+                Text('Headline M', style: theme.headlineMedium),
+                Text('Headline S', style: theme.headlineSmall),
+                Text('Caption XL', style: theme.labelLarge),
+                Text('Caption L', style: theme.labelMedium),
+                Text('Caption M', style: theme.labelSmall),
+                Text('Body L', style: theme.bodyLarge),
+                Text('Body S', style: theme.bodyMedium),
+                Text('Body XS', style: theme.bodySmall),
```

Comment:
1 more type is missing pls confirm with UI/Ux team

Reply:
will do it, once i get the confirmation